src/client/components/Review/Review.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import { useParams, useHistory } from "react-router-dom";
+import FaStar from "react-icons/fa";
+import './review.css'
+import { useMealContext } from "../Context/MealContext";
+const Review = () => {
+    const { currentMeals, dispatchMeals, getMeal } = useMealContext();
```

Comment:
It this displaying a single review or submitting a review? Or both? 
Should it do both or should it be separated?

Reply:
Displaying of a single.meal review.